### ISSUE TITLE
Doc generation fails on Windows

### ISSUE THREAD

**Issue by robstove** [mgechev/angular-seed#1961]
This seems to be only on Windows. I have v10. The root of the issue comes from compodoc but thought I would start here to see if there are changes we can make to the seed.

```..\angular-seed\angular-seed\node_modules\.bin\compodoc:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at createScript (vm.js:74:10)
    at Object.runInThisContext (vm.js:116:10)
    at Module._compile (module.js:533:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "<projpath>\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "compodoc"
```

**Comment by mgechev** [comment]
Hey @robstove, thanks for reporting the issue. This doesn't seem like a problem in angular seed but might be related to the integration with compodoc.

@vogloblinsky can you comment?

**Comment by amalshehu** [comment]
@robstove Once I had this issue with compodoc. I downgraded compodoc dependency version. Then it worked. Can you check the version of compodoc in package.json and the installed version?

**Comment by vogloblinsky** [comment]
@robstove / @amalshehu : which version of Node.js / npm did you use ?

**Comment by amalshehu** [comment]
@vogloblinsky Node:6.10 and npm 3.x

**Comment by robstove** [comment]
@amalshehu I am using the version from the package.json defined in the main repo. https://github.com/mgechev/angular-seed/blob/master/package.json#L51

```"@compodoc/compodoc": "^1.0.0-beta.7",```

@vogloblinsky I started with some flavor of 6 but have since upgraded to 8. What I have found is that there is an issue with Windows and the specific command that is being called.

Similar problem here:
https://github.com/gotwarlost/istanbul/issues/677
